include/jemalloc/internal/flat_bitmap.h

```diff
@@ -0,0 +1,222 @@
+#ifndef JEMALLOC_INTERNAL_FB_H
```

Comment:
Nice abbreviation.

Reply:
lol. It felt weird to keep writing it, but it also felt weird to deliberately pick what seemed like a worse name to avoid the abbreviation.